README.md

```diff
@@ -1 +1,96 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+# Project runs up FastAPI server, which can take data from anilist and do REST API on your own server.
+
+# How install
+_$ sudo apt-get update_
+
+_$ docker pull mongo_
+
+_$ git clone https://github.com/Max2288/rpm-hws_7-1_2023/_
+
+
+# How run
+Add .env file with your data
+
+Go to folder with project and run commands:
+
+_$ docker run -d -p 27018:27017 mongo_
+
+_$ python3.10 -m venv ./venv_
+
+_$ . ./venv/bin/activate_
+
+_$ pip install -r requirements.txt_
+
+# After start
+
+
+## anilist.co
+
+Example of data that we recieve: 
```

Comment:
ANY of requests sent via postman (according to your suggestinons) return WARNING:  Invalid HTTP request received.

Reply:
Give an example of your request